maintainers/scripts/luarocks-packages.csv

```diff
@@ -116,6 +116,7 @@ markdown,,,,,,
 mediator_lua,,,,,,
 middleclass,,,,,,
 mimetypes,,,,,,
+mini.test,,,,,,
```

Comment:
```suggestion
mini.test,,,,,,teto
```

Reply:
I have never used any mini plugin so I dont want to scare away potential maintainers. I do look at issues even when my nickname is not there. The previous vimPlugin had no maintainer either.